android/KMEA/app/src/main/java/com/tavultesoft/kmea/KeyboardPickerActivity.java

```diff
@@ -362,6 +363,18 @@ protected static boolean addKeyboard(Context context, Keyboard keyboardInfo) {
 
       keyboardInfo.setNewKeyboard(true);
       KeyboardController.getInstance().add(keyboardInfo);
+      // Check if "other" keyboards of the same packageID and keyboardID need to update version
+      // Don't use forEach because we might be updating entries
+      for (int i=0; i<KeyboardController.getInstance().get().size(); i++) {
+        Keyboard otherKeyboard = KeyboardController.getInstance().getKeyboardInfo(i);
+        if (otherKeyboard.getPackageID().equals(keyboardInfo.getPackageID())
+            && otherKeyboard.getKeyboardID().equals(keyboardInfo.getKeyboardID())
+            && !otherKeyboard.getLanguageID().equals(keyboardInfo.getLanguageID())
+            && (FileUtils.compareVersions(otherKeyboard.getVersion(), keyboardInfo.getVersion()) == FileUtils.VERSION_LOWER)) {
```

Comment:
I don't think we need the version comparison. There's only one package installed, so we should set all to the incoming version.